posthog/templates/surveys/public_survey.html

```diff
@@ -910,6 +910,7 @@ <h1 class="survey-title">{{ name }}</h1>
             capture_dead_clicks: false,
             capture_exceptions: false,
             advanced_disable_feature_flags: true,
+            persistance: 'memory'
```

Comment:
```suggestion
            persistence: 'memory'
```

can you test the behaviour before going forward?

Reply:
testing and will post on PR description before merging